### DISCUSSION TITLE
Select service server when submitting a request

### DISCUSSION THREAD

**Discussion by dynizzle** [sct/overseerr#680] [Feature Requests]
One of the functionalities I really like about Overseerr is that I can add multiple Sonarr services. It could be expanded on if there was an option when someone requests a show series they could choose which Sonarr service to send it to. That probably would need to have some permissions set behind it for only users that know what they should be choosing.

Maybe if you don't have a default set it prompts a choice of which one to use?

My specific situation is I have two different Sonarr services. One for dubbed anime and one for subbed anime. The reason why that is necessary is there isn't a means in Sonarr to set restrictions on a series-by-series basis. So all the anime it auto downloads will either be subbed or dubbed. So if I want to request a show in Overseerr to either be subbed or dubbed I would need to change the default Sonarr service to match the one I want. It's nice that Overseer lets you at least set a server nickname so I have one called Sonarr Dubbed and the other Sonarr Subbed. 

To get around having to keep changing the default I just straight up have two different instances of Overseerr running, one for the dubbed Sonarr service and one for the subbed Sonarr service.

**Comment by sct** [comment]
This is already possible on the develop branch and will be in the next release.
